scripts/install_third_party_libs_test.py

```diff
@@ -62,6 +63,7 @@ def communicate(self):
             return Ret()
         def mock_popen_error_call(unused_cmd_tokens, *args, **kwargs):
             class Ret(python_utils.OBJECT):
+
```

Comment:
This is still not resolved and the lint test are passing, I don't think your lint test is working as expected! Can you please check what's going on? (Even you made changes in this file in the last commit and I don't know why this wasn't caught during the pre-push checks, can you please explain?)

Reply:
This happened because we have a blank line between the docstring and class name, So our lint check considered it as a class with a docstring and skipped the check for this class.

Reply:
> So our lint check considered it as a class with a docstring and skipped the check for this class.

But there isn't any newline after the docstring and the lint test passes successfully which isn't expected right?

Reply:
Yes, but we need a separate lint to check if there is newline between docstring and class name. As we don't want to have two different lint checks in a single function. This check is working as expected.

Reply:
My question is still persist, in any case do we expect the lint test to fail if there is no newline after the class docstring? I don't think the lint test should pass this:
```
Class A():

    """Some docstring."""
    def __init__():
        ....
```

Reply:
yes, this check do not pass on this one. But I think this for this we need a different lint check to check if the docstring is correctly placed.

Reply:
This check is working as expected because docstring must be placed just after the class name and it is working fine on all those cases.

Reply:
On this commit: https://github.com/oppia/oppia/pull/9687/commits/a3040c70d4cbbb3cd9ae6be7315b072529941eb9 the lint test passes though we have a similar case on around line 66 of `scripts/install_third_party_libs_test.py`, can you please check why the lint test didn't fail locally or on CI?